src/config/babelrc.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+const {hasPkgProp, browserslist} = require('./../utils')
+const useBuiltinBrowserslist = !hasPkgProp('browserslist')
+
+const validateBoolOption = (name, value, defaultValue) => {
+  if (typeof value === 'undefined') {
+    value = defaultValue
+  }
+
+  if (typeof value !== 'boolean') {
+    throw new Error(`Preset aegir: '${name}' option must be a boolean.`)
+  }
+
+  return value
+}
+module.exports = function (api, opts = {}) {
+  const env = process.env.BABEL_ENV || process.env.NODE_ENV
+  const isEnvDevelopment = env === 'development'
+  const isEnvProduction = env === 'production'
+  const isEnvTest = env === 'test'
+  const isFlowEnabled = validateBoolOption('flow', opts.flow, true)
```

Comment:
Couldn't this be just `const isFlowEnabled = opts.flow || true`

Reply:
yes we could, its just validation if the user overrides aegir babelrc and doesnt set flow option to a boolean it gets a nice error in the console.